### DISCUSSION TITLE
Require() ES Module for prettier-plugin-tailwindcs when using external tools?

### DISCUSSION THREAD

**Discussion by Erythros** [tailwindlabs/tailwindcss#12013] [Help] [answered]
Nevermind. Closing this

**Comment by vladmiller** [comment]
Can you please share why "nevermind" and what did you do?

I'm stuck with the same issue in monorepo set-up.

```
"prettier": "3.0.2",
"prettier-plugin-tailwindcss": "^0.5.3",
"tailwindcss": "^3.3.3",
```

**Comment by Erythros** [comment, marked as answer]
@vladmiller because, first of all, I opened it in the wrong place. Secondly, it's not related to the plugin itself, but 3rd party library.

The issue is described here very well: [#214](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/214#issuecomment-1725525854)

Same issue is described [here](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/217#issuecomment-1729893237), with a different but similar solution.

Hope it helps.
